File: testdata/RenameField/Java/current/Employee.java
import java.util.Date;

public class Employee {
    private String employeeName;  // Field renamed from 'name' to 'employeeName'
    private int age;
    private Date joinDate;

    public Employee(String employeeName, int age, Date joinDate) {
        this.employeeName = employeeName;
        this.age = age;
        this.joinDate = joinDate;
    }

    public String getEmployeeName() {  // Getter method renamed to reflect the new field name
        return employeeName;
    }

    public int getAge() {
        return age;
    }

    public Date getJoinDate() {
        return joinDate;
    }
}

File: testdata/RenameField/Cpp/current/Employee.cpp
#include <string>
#include <ctime>

class Employee {
private:
    std::string employeeName;  // Field renamed from 'name' to 'employeeName'
    int age;
    std::tm joinDate;

public:
    Employee(std::string employeeName, int age, std::tm joinDate)
        : employeeName(employeeName), age(age), joinDate(joinDate) {}

    std::string getEmployeeName() {  // Getter method renamed to reflect the new field name
        return employeeName;
    }

    int getAge() {
        return age;
    }

    std::tm getJoinDate() {
        return joinDate;
    }
};
